prisma/schema.prisma

```diff
@@ -6,6 +6,48 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider = "mongodb"
+  url      = env("DPDMONGO_DATABASE_URL")
```

Comment:
is this env var already set or do we need to set it somewhere?

Reply:
it's got to be added to the .env file of the project